### PR TITLE
Exclude examples.build from linter

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM

File: build-system/config.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ var commonTestPaths = [
     served: true
   },
   {
+    pattern: 'examples.build/**/*',
```
Comment (cramforce) [inline build-system/config.js]: Not sure we need this. We served examples for the validator test.

Comment (dvoytenko) [inline build-system/config.js]: Removed

**Pull request merged**